rclcpp/package.xml

```diff
@@ -30,7 +30,7 @@
   <build_export_depend>rosidl_typesupport_cpp</build_export_depend>
 
   <depend>libstatistics_collector</depend>
-  <depend>rcl</depend>
+  <depend version_gte="5.3.4">rcl</depend>
```

Comment:
https://github.com/ros2/rclcpp/commit/52327dd3a3b0fc47446ce86ecaf16fdde4fc4a60 is rclcpp version, so this should be the correct version.